### PR TITLE
Update Ruby to 2.2

### PR DESCRIPTION
**Pull request by AEtherC0r3**
* Force development environment to use ruby 2.2 in preparation for the update to rails 5
* Update vagrant box image to use openSUSE 42.2
* Fix dependencies

Related to #1087

### CODE REVIEW

**Comment by differentreality** [comment]
Anyone wants to review this in order to start getting Rails 5 updates merged?

**Comment by Ana06** [comment]
@AEtherC0r3 

> Update vagrant box image to use openSUSE 42.2

How is this related to Update Ruby to 2.2?

**Comment by AEtherC0r3** [comment]
@Ana06 It isn't. But, the 42.1 release has been EOLed since May, so I thought that it was a good idea to update the base image, along with the Ruby update

**Comment by Ana06** [comment]
@AEtherC0r3

> @Ana06 It isn't. But, the 42.1 release has been EOLed since May, so I thought that it was a good idea to update the base image along with the Ruby update

Then put it in a separate commit. :wink:





> @Ana06 It isn't. But, the 42.1 release has been EOLed since May, so I thought that it was a good idea to update the base image along with the Ruby update

Then put it in a separate commit :wink:

**Comment by AEtherC0r3** [comment]
@Ana06 Done :)

File: bootstrap.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ if [ ! -f /vagrant/config/database.yml ] && [ -f /vagrant/config/database.yml.ex
     echo -e "WARNING: Please make sure this database works in this vagrant box!\n\n"
   fi
 else
-  echo -e "\nnWARNING: You have already configured your database in config/database.yml."
+  echo -e "\n\nWARNING: You have already configured your database in config/database.yml."
```
Comment (Ana06) [inline bootstrap.sh]: This is also not related with the ruby update, so I would also put it in a different commit. :wink:

**Comment by differentreality** [comment]
:+1: Looking forward to seeing more updates (gems, Rails 5 and such).
